code/modules/research/nanites/nanite_programs/healing.dm

```diff
@@ -48,16 +48,10 @@
 
 /datum/nanite_program/purging
 	name = "Blood Purification"
-	desc = "The nanites purge toxins and chemicals from the host's bloodstream."
+	desc = "The nanites purge toxins and chemicals from the host's bloodstream. Consumes nanites even if it has no effect."
```

Comment:
tbh, it wouldn't be so bad if blood purification and selective blood purification had a sort of accelerated regeneration/bio-reconstruction dynamic, where blood purification would self-regulate (but be unable to tell toxins from medicines), while selective blood purification would need to be told when to activate (but be able to avoid purging non-toxins from the bloodstream).

Reply:
The issue is that blood purification also has a permenant 1 toxin healing aswell, so its well worth just putting it in anyways.

Reply:
Won't it consume nanites, though?